pangram/src/test/java/PangramTest.java

```diff
@@ -0,0 +1,38 @@
+import org.junit.Test;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+
+public class PangramTest {
+
+    @Test
+    public void emptySentence() {
+        Pangram pangram = new Pangram("");
```

Comment:
Not sure what the other tracks look like, but I think it's odd to return an object of type `Pangram` that is .. not a pangram.

In terms of Java, I'd recommend instead something like
```java
@Test(expected = IllegalArgumentException.class)
public void itRejectsEmptySentence() {
    Pangram p = Pangram.of("");
}
```
Or if you want to stick with an `isPangram` method, I'd stick with a static method on a `Pangrams` helper class.
```java
@Test
public void itRejectsEmptySentence() {
{
    assertFalse(Pangrams.isPangram(""));
}
```